web_programming/instagram_pic.py

```diff
@@ -1,37 +1,68 @@
 from datetime import datetime
+from typing import Optional
```

Comment:
Not needed on Python 3.12
```suggestion

```

Reply:
OK

Reply:
@cclauss tried many times but every time getting this error
Error: web_programming/instagram_pic.py:1:1: I001 Import block is un-sorted or un-formatted

Reply:
If the imports were working fine before, then do not change them.

Click the text __I001__ to read the rule and how it works.

`python3 -m pip install ruff` and then `ruff --select=I --fix web_programming/instagram_pic.py`